src/thunderbots/software/grsim_communication/visitor/grsim_command_primitive_visitor.cpp

```diff
@@ -72,7 +72,34 @@ void GrsimCommandPrimitiveVisitor::visit(const CatchPrimitive &catch_primitive)
 
 void GrsimCommandPrimitiveVisitor::visit(const ChipPrimitive &chip_primitive)
 {
-    // TODO: https://github.com/UBC-Thunderbots/Software/issues/93
+    // Determine if robot is at least 1 robot radius from the given location
+    if ((chip_primitive.getChipOrigin() - robot.position()).len() >= 1)
+    {
+        // Travel in straight line from current position to given location
+        motion_controller_command = MotionController::MotionControllerCommand(
+            chip_primitive.getChipOrigin(), chip_primitive.getChipDirection(), 0.0,
+            chip_primitive.getChipDistance(), true, false);
+
+        // Move in a straight line directly behind the robot
+        Point destBehind;
+        destBehind = chip_primitive.getChipOrigin();
+        destBehind = Point(destBehind.x(), destBehind.y());
+        destBehind.set(destBehind.x() - 0.5, destBehind.y() - 0.5);
```

Comment:
So the robot will always move from  `(chip_primitive.getChipOrigin()-0.5,chip_primitive.getChipOrigin()-0.5)` to `(chip_primitive.getChipOrigin()+0.8,chip_primitive.getChipOrigin()+0.8)` no matter the direction we are chipping? It seems to me that robot is moving in only one direction no matter which direction we are chipping the ball.